docs/querying/raw-queries.mdx

```diff
@@ -293,6 +321,41 @@ Read more about this in [#15142](https://github.com/sequelize/sequelize/issues/1
 
 :::
 
+### `sql.join`
+
+The `sql.join` function can be used to join multiple SQL fragments together.
+It is designed to be the equivalent of [`Array.prototype.join`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join) for SQL fragments.
+
+```ts
+const columns = [sql.identifier('name'), sql.identifier('funding')];
+
+await sequelize.query(sql`SELECT ${sql.join(columns, ', ')} FROM projects`);
+```
+
+```sql
+SELECT "name", "funding" FROM projects
```

Comment:
`-- The identifier quotes are dialect-specific, this is an example for PostgreSQL`
Question; do we want to copy this line for these examples as well? I think they're less useful here, but interested to hear what you think

Reply:
Done!
Better to be consistent